src/main/target/DIAT_MAMBAF405_2022B/target.h

```diff
@@ -0,0 +1,154 @@
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "DIAT"
+#define USBD_PRODUCT_STRING "MAMBAF405_2022B"
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC_SPI_MPU6500
+#define USE_GYRO_SPI_MPU6500
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASHFS
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+#define USE_BARO
+#define USE_ADC
+#define USE_SPI_GYRO
+// manual
+#define USE_VCP
+#define USE_SERIAL_4WAY_BLHELI_INTERFACE
+
+#define USE_LED
+#define LED0_PIN             PC15
+#define LED1_PIN             PC14
+#define BEEPER_PIN           PC13
+#define BEEPER_INVERTED
+
+#define USE_SPI
+#define USE_SPI_DEVICE_1
+#define SPI1_SCK_PIN         PA5
+#define SPI1_MOSI_PIN        PA6
+#define SPI1_MISO_PIN        PA7
+#define USE_SPI_DEVICE_2
+#define SPI2_SCK_PIN         PB13
+#define SPI2_MOSI_PIN        PB14
+#define SPI2_MISO_PIN        PB15
+#define USE_SPI_DEVICE_3
+#define SPI3_SCK_PIN         PC10
+#define SPI3_MOSI_PIN        PC11
+#define SPI3_MISO_PIN        PB5
+
+#define USE_EXTI
+
+#define GYRO_1_ALIGN CW270_DEG
+#define GYRO_1_CS_PIN        PA4
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_1_SPI_INSTANCE SPI1
+
+#define USE_MPU_DATA_READY_SIGNAL
+
+#define ACC_MPU6000_ALIGN         GYRO_1_ALIGN
+#define GYRO_MPU6000_ALIGN        GYRO_1_ALIGN
+#define MPU6000_CS_PIN            GYRO_1_CS_PIN
+#define MPU6000_SPI_INSTANCE      GYRO_1_SPI_INSTANCE
+#define MPU_INT_EXTI              GYRO_1_EXTI_PIN
+
+#define ACC_MPU6500_ALIGN         GYRO_1_ALIGN
+#define GYRO_MPU6500_ALIGN        GYRO_1_ALIGN
+#define MPU6500_CS_PIN            GYRO_1_CS_PIN
+#define MPU6500_SPI_INSTANCE      GYRO_1_SPI_INSTANCE
+#define MPU_INT_EXTI              GYRO_1_EXTI_PIN
+
+#define ACC_ICM42688P_ALIGN       GYRO_1_ALIGN
+#define GYRO_ICM42688P_ALIGN      GYRO_1_ALIGN
+#define ICM42688P_CS_PIN          GYRO_1_CS_PIN
+#define ICM42688P_SPI_INSTANCE    GYRO_1_SPI_INSTANCE
+
+// notice - this file was programmatically generated and may need GYRO_2 manually added.
+
+#define USE_I2C
+#define USE_I2C_DEVICE_1
+#define MAG_I2C_INSTANCE (I2CDEV_1)
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+#define I2C1_SCL PB8
+#define I2C1_SDA PB9
+
+#define FLASH_CS_PIN         PA15
+#define FLASH_SPI_INSTANCE SPI3
+#define ENABLE_BLACKBOX_LOGGING_ON_SPIFLASH_BY_DEFAULT
+
+#define MAX7456_SPI_CS_PIN   PB12
+#define MAX7456_SPI_INSTANCE SPI2
+
+#define UART1_TX_PIN         PB6
+#define UART2_TX_PIN         PA2
+#define UART3_TX_PIN         PB10
+#define UART4_TX_PIN         PA0
+#define UART5_TX_PIN         PC12
+#define UART6_TX_PIN         PC6
+#define UART1_RX_PIN         PB7
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PB11
+#define UART4_RX_PIN         PA1
+#define UART5_RX_PIN         PD2
+#define UART6_RX_PIN         PC7
+#define INVERTER_PIN_UART1   PC0
```

Comment:
Diatone Mamba wiring diagram and manual lists the S/F.Port as being UART5
![image](https://github.com/emuflight/EmuFlight/assets/85623381/801ff58a-49ec-4de3-8c98-446aa237b619)

Reply:
so `#define RSSI_ADC_PIN        PC12`  ???